gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/root/resources/endpoint/login/LoginCallbackEndpoint.java

```diff
@@ -37,7 +37,10 @@
     @Override
     public void handle(RoutingContext routingContext) {
         final Session session = routingContext.session();
-        final String returnURL = UriBuilderRequest.resolveProxyRequest(routingContext.request(), routingContext.get(CONTEXT_PATH) + "/oauth/authorize", (MultiMap) routingContext.get(PARAM_CONTEXT_KEY), true);
+        final String returnURL =
```

Comment:
Maybe we can move this URL builder step to a common method into the AbstractEndpoint class in order to avoid missing implementation if we have to change something? wdyt?